.github/workflows/ci.yaml

```
@@ -21,7 +21,7 @@ jobs:
         with:
           python-version: '3.10'
       - uses: abatilo/actions-poetry@v2
-      - run: sudo apt-get install -y rdfind symlinks
+      - run: sudo apt-get install -y rdfind symlinks patchutils
```

❓ Is this a stray change from the diff effort?
```suggestion
      - run: sudo apt-get install -y rdfind symlinks
```